### DISCUSSION TITLE
GitHub student benefits rejected due to university email

### DISCUSSION THREAD

**Discussion by AJ-cyber** [community/community#63161] [GitHub Education]
### Hi everyone,

I have successfully applied for the GitHub student benefits twice already when I was in high school. I recently graduated and my membership also expired. I am currently in University studying computer science and I wish to reapply for the student benefit program. But my application has been denied twice due to the email used for my application, the university I attend does not issue an educational email. We are only given portal login information and we use our personal email and phone number for everything else. 
Please what do I do?

**Comment by humairshoukat** [comment]
try submitting your university student ID card. try submitting it from your phone. try taking the picture while submitting it.
